Mathlib/Data/Finset/Sym.lean

```diff
@@ -2,9 +2,8 @@
 Copyright (c) 2021 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
-
-[`data.finset.sym`@`98e83c3d541c77cdb7da20d79611a780ff8e7d90`..`02ba8949f486ebecf93fe7460f1ed0564b5e442c`](https://leanprover-community.github.io/mathlib-port-status/file/data/finset/sym?range=98e83c3d541c77cdb7da20d79611a780ff8e7d90..02ba8949f486ebecf93fe7460f1ed0564b5e442c)
 -/
+
```

Comment:
I think we tend to *not* have the line break

Reply:
Right; let me fix that quickly.

Reply:
To be more precise: the style guidelines advice against it, and mathlib is sufficiently un-ambiguous that this PR is not the place to litigate the question.